.gitignore

```diff
@@ -1,5 +1,6 @@
-out/
-dev/
+artifacts/
 node_modules/
+out/
+wbpkd/
+.dist/
 .vscode-test/
```

Comment:
let's include `.idea` directory either, so JetBrains IDE users (like myself :wink:) are not harmed.

Reply:
@P0lip Fixed